frontend/src/metabase/visualizations/components/ChoroplethMap.jsx

```diff
@@ -110,10 +110,19 @@ const mapStateToProps = state => ({
 
 const connector = connect(mapStateToProps, null);
 
-function getMapUrl(details, props) {
+const addTrailingSlash = url => (url.endsWith("/") ? url : url + "/");
```

Comment:
nit: A little misleading. This function only adds a trailing slash for URLs without it. And I might be worried to pass a URL with a trailing slash to this function.

I'd name this

```suggestion
const ensureTrailingSlash = url => (url.endsWith("/") ? url : url + "/");
```